test/check-application

```diff
@@ -51,13 +51,88 @@ class TestApplication(testlib.MachineCase):
         # show container listing toggle
         b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
         b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
-
+        #show running container
         b.click("#containers-containers-filter button")
         b.wait_present("#containers-containers-filter .dropdown-menu")
         b.click("#containers-containers-filter li[data-data='running'] a")
+        b.wait_present('#containers-containers tr:contains("busybox:latest")')
+        b.wait_visible('#containers-containers tr:contains("busybox:latest") + tr:contains("sleep 1000")')
+
+        # switch to all containers
+        b.wait_present("#containers-containers-filter button")
+        b.click("#containers-containers-filter button")
+        b.wait_visible("#containers-containers-filter .dropdown-menu")
+        b.click("#containers-containers-filter li[data-data='all'] a")
+        b.wait_present("#containers-containers")
+        b.wait_present('#containers-containers tr:contains("busybox:latest") td:contains("running")')
+        b.wait_present('#containers-containers tr:contains("alpine:latest")')
+
+        # delete a stopped container
+        b.click('#containers-containers tbody tr:contains("alpine:latest") td.listing-ct-toggle')
+        b.wait_visible('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        b.click('#containers-containers tbody tr:contains("alpine:latest") + tr button.btn-delete')
+        with b.wait_timeout(60):
+            b.wait_present(".modal-dialog div")
+        b.wait_visible(".modal-dialog div .btn-danger")
+        b.click(".modal-dialog div .btn-danger")
+        with b.wait_timeout(60):
+            try:
+                b.wait_visible("#containers-containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+            except:
+                self.confirm()
+                b.wait_visible("#containers")
+                b.wait_not_in_text("#containers-containers", "alpine:latest")
+
+        # delete a running container
+        self.loadAllContainerlist()
```

Comment:
This should not need a page reload here.

Reply:
then what should be here? without 
```
b.reload() 
b.enter_page("/podman")
```
test failed.
but in my code for podman UI, I only rerender the component to update after deletion, I didn't use reload.

Reply:
Doesn't that point out a bug in the code then? For all actions that you do in the UI, the UI should update automatically. You only need to reload the page for actions that you do outside, like with executing `podman` commands.

Reply:
```
        # self.pageReload()
        self.loadAllContainerlist()
        b.wait_present("#containers-containers")
        b.wait_present('#containers-containers tbody tr:contains("busybox:latest")')
        b.click('#containers-containers tbody tr:contains("busybox:latest") td.listing-ct-toggle')
        b.wait_present('#containers-containers tbody tr:contains("busybox:latest") + tr button.btn-delete')
```
without self.pageReload(), tests hangs in wait_present ```tr button.btn-delete```, what kind of bug might be?